### PR TITLE
Add a short guide to using initrd_path

### PR DESCRIPTION
**Pull request by jeromegn**
## Reason for This PR

I just tried using `initrd_path` and had to get more information in the firecracker Slack because some things were unclear due to the lack of documentation.

## Description of Changes

Added a very short guide to creating an initrd and using `initrd_path`.

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license.

## PR Checklist

- [x] All commits in this PR are signed (`git commit -s`).
- [x] The reason for this PR is clearly provided (issue no. or explanation).
- [x] The description of changes is clear and encompassing.
- [x] Any required documentation changes (code and docs) are included in this PR.
- [x] Any newly added `unsafe` code is properly documented.
- [x] Any API changes are reflected in `firecracker/swagger.yaml`.
- [x] Any user-facing changes are mentioned in `CHANGELOG.md`.
- [x] All added/changed functionality is tested.

### CODE REVIEW

File: docs/initrd.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Creating and using an initrd for Firecracker
+
+## Creating
+
+### Based on alpine or suse
+
+You can use this script to generate an initrd either based on alpine or suse linux: https://github.com/marcov/firecracker-initrd
+
+The script extracts
```
Comment (dianpopa) [inline docs/initrd.md:9]: This sentence seems incomplete.

**Review by dianpopa — COMMENTED** [review]
Thanks for this contribution! There is only one nit you need to solve and then it is good to merge.

File: docs/initrd.md (hunk 1 above)
Comment (jeromegn) [reply docs/initrd.md]: Just finished the sentence. I think that's what I intended to write :)

**Review by dianpopa — APPROVED** [review]

**Review by luminitavoicu — CHANGES_REQUESTED** [review]
Guide LGTM. Maybe you could squash the two commits into one. It doesn't really make sense to have a separate commit for finishing the sentence.

**Comment by jeromegn** [comment]
I squashed my commits and rebased master.

**Review by dianpopa — APPROVED** [review]

**Review by luminitavoicu — APPROVED** [review]

**Pull request merged**